src/Microsoft.AspNet.Mvc.Core/ApiExplorer/IApiResponseFormatMetadataProvider.cs

```diff
@@ -1,8 +1,8 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-using System;
 using System.Collections.Generic;
+using Microsoft.AspNet.Mvc.Formatters;
```

Comment:
This is a bit iffy because Roslyn doesn't detect the namespace is used in doc comments. Will trip up the next person who removes supposedly-unused `using`s.